conf/messages

```diff
@@ -772,6 +772,14 @@ benefits.entertainmentBenefitAmount.error.overMaximum.agent = Your client’s en
 benefits.entertainmentBenefitAmount.error.invalidFormat.individual = Enter the amount you got for entertainment in the correct format
 benefits.entertainmentBenefitAmount.error.invalidFormat.agent = Enter the amount your client got for entertainment in the correct format
 
+benefits.incidentalOvernightCostEmploymentBenefits.title.individual = Did you get any incidental overnight costs?
+benefits.incidentalOvernightCostEmploymentBenefits.title.agent = Did your client get any incidental overnight costs?
+benefits.incidentalOvernightCostEmploymentBenefits.thisIncludes.individual = These are personal costs you incurred while travelling overnight on business.
+benefits.incidentalOvernightCostEmploymentBenefits.thisIncludes.agent = These are personal costs they incurred while travelling overnight on business.
```

Comment:
_thisIncludes_ and _thisIncludes2_ could perhaps be better named

They aren't really an includes message 

perhaps something like

benefits.incidentalOvernightCostEmploymentBenefits.costInfo
benefits.incidentalOvernightCostEmploymentBenefits.allowanceInfo